openmc/mgxs/mgxs.py

```diff
@@ -149,6 +149,8 @@ class MGXS(object):
         Name of the multi-group cross section
     rxn_type : str
         Reaction type (e.g., 'total', 'nu-fission', etc.)
+    nu : bool
```

Comment:
Ok I can see how you've made the docstrings "hide" these new properties from those classes for which they don't apply. However, when a user inspects the available properties on an `MGXS` subclass using tab completion in IPython or Jupyter Notebook, they will see `nu` and `prompt` available for each one. I hope I'm not being too needy in asking you to move these properties to the relevent subclasses, even at the cost of a little redundant code (this PR will still easily be a net negative in terms of code reduction). Otherwise, I know for a fact that I'll be explaining this design decision away to confused users, and I want to avoid having to point to your (3) above as the reason :-)